inst/config/minimal.yaml

```diff
@@ -3,7 +3,8 @@ title: "fastTestRun"
 basedOn: "default.yaml"
 # scope and resolution
 periods: [2000, 2005, 2010, 2015, 2020, 2030, 2040]
+regionmapping: "input/regionmappingEUplus.csv"
 regions: DEU
 # data
-startingPoint: "output/small_2023-04-17_11.04.00/abort.gdx"
```

Comment:
As a config shipped with the package, it shouldn't reference a file that cannot be found. My bad, I know. But couldn't we just remove the starting point?

Reply:
I agree, and yes, I'd say we can remove the starting point.